acceptance/bundle/paths/pipeline_globs/output.txt

```diff
@@ -0,0 +1,9 @@
+
+>>> [CLI] bundle validate --output json
+Error: resources/pipeline.yml defined in 'include' section does not match any files
```

Comment:
I don't know if I am missing something but I don't think it provides coverage for the regression. You'll need to correct the path in include.